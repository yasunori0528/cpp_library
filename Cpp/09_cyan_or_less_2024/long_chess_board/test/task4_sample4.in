2
3 987654321
1 159265358 2 323846264
3 876543210
3 209749445 1 781640628
