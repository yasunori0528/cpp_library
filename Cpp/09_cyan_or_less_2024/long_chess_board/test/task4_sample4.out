82290453
285945592
